assets/javascripts/discourse/connectors/composer-action-after/encrypt.js.es6

```diff
@@ -10,10 +10,24 @@ export default {
   setupComponent(args, component) {
     const currentUser = getOwner(component).lookup("current-user:main");
     const status = getEncryptionStatus(currentUser);
+    const timerOptions = {
+      "3_minutes": "3",
+      "1_hour": "60",
+      "3_hours": "180",
+      "12_hours": "720",
+      "24_hours": "1440",
+      "3_days": "4320",
+      "7_days": "10080",
+    };
 
     component.setProperties({
       isEncryptEnabled: status !== ENCRYPT_DISABLED,
       isEncryptActive: status === ENCRYPT_ACTIVE,
+      timeBombRequired: this.siteSettings.require_time_bombs,
+      timeBombLength: this.siteSettings.required_time_bomb_length.replace(
```

Comment:
This is a bit unusual. Make we should have the `timerOptions` contain the spaces?